src/collections/_documentation/clients/go/index.md

```diff
@@ -31,9 +31,10 @@ Raven-Go can be installed like any other Go library through `go get`:
 $ go get github.com/getsentry/raven-go
 ```
 
-## Configuring the Client {#configure}
+## Configuration {#configure}
 
 To use `raven-go`, you’ll need to import the `raven` package, then initilize your DSN globally. If you specify the `SENTRY_DSN` environment variable, this will be done automatically for you. The release and environment can also be specified in the environment variables `SENTRY_RELEASE` and `SENTRY_ENVIRONMENT` respectively.
```

Comment:
Little typo on "initialize." Here's the edit, "To use raven-go, you’ll need to import the raven package, then initialize your DSN globally."